lib/bolt/plugin/module.rb

```diff
@@ -30,6 +30,19 @@ def initialize(mod:, context:, config:, **_opts)
         @module = mod
         @config = config
         @context = context
+
+        # Raise a deprecation warning if the pkcs7 plugin is using hyphenated keys
+        if @module.name == 'pkcs7'
+          %w[private-key public-key].each do |key|
+            next unless @config.key?(key)
+            fixed_key = key.gsub('-', '_')
+            @config[fixed_key] = @config.delete(key)
+
+            message = "pkcs7 key #{key} is deprecated and will be removed in a future version "\
+                      "of Bolt; use #{fixed_key} instead."
+            Logging.logger[self].warn(message)
+          end
+        end
```

Comment:
There are a few different places this can happen, including `Bolt::Plugin.add_module_plugin` and `Bolt::Plugin.config_for_plugin`. I stuck it here since it seemed to be the most-specific to the plugin itself before validating the config.

Reply:
Can these be passed to a specific plugin invocation? What happens in that case?

Reply:
Ah, that will still error out. Updated to handle it in both locations.